README.md

```diff
@@ -9,26 +9,26 @@ This project is made using Django 2.1 for backend.
 1. Create a virtual environment and activate it.
 
 ```console
-$ python3 -m venv hack
-$ source hack/bin/activate
+python3 -m venv hack
+source hack/bin/activate
 ```
 
 2. Install dependencies.
 
 ```console
-$ pip install -r requirements.txt
+pip install -r requirements.txt
 ```
 
 3. Create the database.
 
 ```console
-$ python manage.py migrate
+python3 manage.py migrate
```

Comment:
I don't think this change is necessary. In virtual environment, only the python version you used to make it is available.